src/main/java/com/google/sps/Utility.java

```diff
@@ -232,4 +235,37 @@ public static MultiMutation filterMultiMutationByNodes(MultiMutation mm, Set<Str
         .setReason(mm.getReason())
         .build();
   }
+
+  /**
+   * Given a list of node names, a map from node name to mutation indices of that node and a list of
+   * multimutations applied to all nodes, returns a list of indices of multimutations in which any
+   * of the nodes in nodeNames get mutated (returned in sorted order)
+   *
+   * @param nodeNames the names of nodes to restrict the returned list of mutations to
```

Comment:
or if you reeeaaaaally wanted to include the "", consider adding that into the comment?

Reply:
Haha no I think you're right